app/js/app-menu.js

```diff
@@ -258,9 +284,27 @@ module.exports = function() {
       }
     });
 
+    $('#import-celldesigner-file').click(function (){
+        $("#celldesigner-file-input").trigger('click');
+    });
+
+    $('#celldesigner-file-input').change(function (e, fileObject) {
+
+      if ($(this).val() != "" || fileObject) {
+        var file = this.files[0] || fileObject;
+        document.getElementById('file-name').innerHTML = file.name;
```

Comment:
@gokberkkaraca no need for this, when the `sbgnvizLoadFileEnd` is triggered properly.